inheritance/src/test/java/inheritance/LibraryTest.java

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
You can delete the generic Library and LibraryTest classes that grandle init creates